pkg/plugins/trigger/trigger.go

```diff
@@ -76,23 +76,22 @@ var plugin = plugins.Plugin{
 }
 
 func init() {
-	customTriggerCommand := os.Getenv(customerTriggerCommandEnvVar)
-	if customTriggerCommand != "" {
-		customCommand := plugins.Command{
-			Name: customTriggerCommand,
-			Arg: &plugins.CommandArg{
-				Pattern: `[-\w]+(?:,[-\w]+)*`,
-			},
-			Description: fmt.Sprintf("Manually trigger /%s chatops commands.", customTriggerCommand),
-			Featured:    true,
-			Action: plugins.
-				Invoke(handleGenericCommentEvent).
-				When(plugins.Action(scm.ActionCreate), plugins.IsPR(), plugins.IssueState("open")),
-		}
-		plugin.Commands = append(plugin.Commands, customCommand)
-	}
-
-	plugins.RegisterPlugin(pluginName, plugin)
+  customTriggerCommand := os.Getenv(customerTriggerCommandEnvVar)
+  for _, trigger := range strings.Split(customTriggerCommand, ","){
```

Comment:
Can you not already do this using a regex pattern like: `build|plan|apply`?

Reply:
Yeah, I agree. This is interpreted as a regexp, so I think you should revert this change and update the documentation and tests accordingly.